app/graphql/mutations/create_route_location.rb

```diff
@@ -0,0 +1,25 @@
+module Mutations
+  class CreateRouteLocation < BaseMutation
+
+    field :route_location, Types::RouteLocationType, null: false
+    field :errors, [String], null: false
+
+    argument :location_id, Types::LocationType, Integer, required: true
+    argument :route_id, Types::RouteType, Integer, required: true
+
+    def resolve(route_id:, location_id:, route_location:)
+      routeLocation = RouteLocation.save(route_id: route_id, location_id: location_id)
+      if RouteLocation.save
+        {
+          routeLocation: routeLocation,
+          errors: []
+        }
+      else
+        {
+          routeLocation: nil,
+          errors: routeLocation.errors.full_messages
```

Comment:
It might be worth updating all `routeLocation` names to 'route_location` in this file. Looks like on your fields you reference it as `route_location`

Reply:
I've updated those to be route_locations and I'm still getting the same error. Am I supposed to be showing you the new code or just commenting like this?

Reply:
After you update code make a new commit and push so it will update here.